ruby/vampires.rb

```diff
@@ -0,0 +1,52 @@
+puts "How many employees are you processing?"
+num_employees = gets.chomp.to_i
+
+num_employees.times do
+		
+	puts "What is your name?"
+	employee = gets.chomp
+	puts "How old are you?"
+	age = gets.chomp.to_i
+	puts "What year were you born?"
+	birthyear = gets.chomp.to_i
```

Comment:
```age_correct = 2017 - age == birth_year```

Couple things here - when we compare the two integers we are going to get true or false as a return, therefore we don't need to say, if this things gives me true set this thing to true, rather set this thing to what this expression evaluates to.  Also, snake case the variables for readability.